tests/tests-negative-scenarios.sh

```diff
@@ -53,7 +53,8 @@ test_pipeline_execution_fails() {
 
 test_pipeline_execution_is_unstable() {
   run_jfr_docker_image "$jenkinsfile_runner_valid_tag" "$current_directory/test_resources/negative-scenarios/test_pipeline_execution_is_unstable/Jenkinsfile"
-  jenkinsfile_execution_should_be_unstable "$?"
+  execution_success "$?"
```

Comment:
Isn't this checking the same?

Reply:
`jenkinsfile_execution_should_be_unstable` checks one more content in the logs, but after checking some executions in PRs and master, I've seen it could cause some flakiness, so I feel more confortable if we prevent that.

Reply:
Fair enough!